training/SYMBOL_TABLE_OVERVIEW.md

```diff
@@ -0,0 +1,150 @@
+# Overview
+
+The purpose of this document is to serve as a high-level guide to the structure and elements within Arkouda's Symbol Table. This guide does not cover creating or refactoring new Symbol Table Entry types. 
+
+<a id="toc"></a>
+## Table of Contents
+1. [Symbol Table Overview](#overview)
+2. [Symbol Table Structure](#symStruct)
+3. [Creating a New Entry From an Array](#addSym)
+4. [Finding an Entry in the Symbol Table](#findSym)
+5. [Accessing Symbol Table Entry Data](#accessSym)
+
+<a id="overview"></a>
+## Symbol Table Overview
+
+The Symbol Table is Arkouda's "database" that tracks the components every Arkouda object is composed of. At a fundamental level it functions similar to a relational database, however the design and access methods are quite different.

```

Comment:
I second this because I have no clue what a relational database and that would make me feel like it was a complex object that I couldn't understand. I think lookup table feels more approachable

Reply:
I reworded this to use `lookup table` and mention that it uses a Chapel `Map`